src/styles/styles.js

```diff
@@ -266,6 +266,11 @@ const styles = {
         borderWidth: 0,
     },
 
+    buttonDisabledBG: {
+        backgroundColor: themeColors.border,
```

Comment:
Can you please make a new themeColor for this? You should add a line that has `buttonDisabledBG: colors.gray2,` at line 21 of `src/styles/themes/default.js` and then reference `themeColors.buttonDisabledBG` here in `styles.js`

Reply:
Changes done.